dotcom-rendering/src/web/layouts/NewsletterSignupLayout.tsx

```diff
@@ -242,38 +336,150 @@ export const NewsletterSignupLayout = ({ CAPIArticle, NAV, format }: Props) => {
 						</span>
 					</div>
 				</ContainerLayout>
+
 				<ContainerLayout
 					centralBorder="full"
 					sideBorders={true}
-					title="UK Focused"
-					description={CAPIArticle.headline}
+					stretchRight={true}
+					leftContent={<NewsletterDetail text="UK Focused" />}
 				>
-					<ArticleContainer format={format}>
-						<ArticleBody
-							format={format}
-							blocks={CAPIArticle.blocks}
-							adTargeting={adTargeting}
-							host={host}
-							pageId={CAPIArticle.pageId}
-							webTitle={CAPIArticle.webTitle}
-							ajaxUrl={CAPIArticle.config.ajaxUrl}
-							switches={CAPIArticle.config.switches}
-							isSensitive={CAPIArticle.config.isSensitive}
-							isAdFreeUser={CAPIArticle.isAdFreeUser}
-							section={CAPIArticle.config.section}
-							shouldHideReaderRevenue={
-								CAPIArticle.shouldHideReaderRevenue
-							}
-							tags={CAPIArticle.tags}
-							isPaidContent={!!CAPIArticle.config.isPaidContent}
-							contributionsServiceUrl={contributionsServiceUrl}
-							contentType={CAPIArticle.contentType}
-							sectionName={CAPIArticle.sectionName || ''}
-							isPreview={CAPIArticle.config.isPreview}
-							idUrl={CAPIArticle.config.idUrl || ''}
-							isDev={!!CAPIArticle.config.isDev}
-						/>
-					</ArticleContainer>
+					<Columns collapseUntil="desktop">
+						<Column width={[1, 1, 5 / 8, 1 / 2, 1 / 2]}>
+							<NewsletterDetail
+								text="UK Focused"
+								showUntil="leftCol"
+							/>
+
+							<ArticleHeadline
+								format={format}
+								headlineString={CAPIArticle.headline}
+								tags={CAPIArticle.tags}
+								byline={CAPIArticle.author.byline}
+								webPublicationDateDeprecated={
+									CAPIArticle.webPublicationDateDeprecated
+								}
+							/>
+
+							<Standfirst
+								format={format}
+								standfirst={CAPIArticle.standfirst}
+							/>
+
+							{/* TODO:
+								- This data will come from the Newsletters API
+								- Only render this part if preview link exists
+								- Add onClick handler or link?
+								- Accessibility?
+							*/}
+							{showNewsletterPreview && (
+								<div css={previewButtonWrapperStyle}>
+									<Button
```

Comment:
If this is a plain link, we can use a `LinkButton` here. That way we won't need javascript 🎉